### PR TITLE
check for cuda nvcc and found in file version

### PR DESCRIPTION
**Pull request by bissenbay**
## Related Issues and Dependencies

https://github.com/thoth-station/user-api/issues/794

## This introduces a breaking change

- [ ] Yes
- [x] No

## This should yield a new module release

- [ ] Yes
- [x] No

## This Pull Request implements

Gets cuda version found in file and from nvcc command and compare both.

### CODE REVIEW

**Comment by bissenbay** [comment]
@fridex I tested locally and the result package-run for the image `quay.io/thoth-station/image-store:cuda-44324038910ae778b3d15bd2ea80148c8ddcb95f310ccabe6f0499003c0425bd` got synced.

**Review by pacospace — COMMENTED** [review]
lgtm

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -4198,7 +4199,7 @@ def sync_analysis_result(self, document: dict) -> None:
                 image_sha=document["result"]["layers"][-1],
                 os_name=os_name,
                 os_version=os_version,
-                cuda_version=cuda_version,
+                cuda_version=cuda_nvcc_version,
```
Comment (fridex) [inline thoth/storages/graph/postgres.py:4202]: What about saving the one found?

```suggestion
                cuda_version=cuda_nvcc_version or cuda_found_in_file_version,
```

Comment (bissenbay) [reply thoth/storages/graph/postgres.py]: Nice, thanks @fridex !

**Review by fridex — APPROVED** [review]
Nice, thanks! :100:

**Pull request merged**